DataProcessing/datax-host/src/main/scala/datax/sink/BlobSinker.scala

```diff
@@ -157,8 +164,16 @@ object BlobSinker extends SinkOperatorFactory {
     if(formatConf.isDefined && !formatConf.get.equalsIgnoreCase("json"))
       throw new Error(s"Output format: ${formatConf.get} as specified in the config is not supported")
     val outputFolders = blobOutputConf.groups.map{case(k,v)=>k->KeyVaultClient.resolveSecretIfAny(v.folder)}
-
-    val jsonSinkDelegate = (rowInfo: Row, rows: Seq[Row], outputPartitionTime: Timestamp, partitionId: Int, loggerSuffix: String) => {
+    
+	val sparkConf = ConfigManager.initSparkConf
+    val spark = SparkSessionSingleton.getInstance(sparkConf)
+    val sc = spark.sparkContext
+	val sa = getStorageAccountName(outputFolders.head._2)
```

Comment:
Do we need any null check for these new lines above?

Reply:
There will always be atleast one value for output folder